test/regressions/webpack.config.js

```diff
@@ -24,6 +24,10 @@ module.exports = {
     new webpack.DefinePlugin({
       DISABLE_CHANCE_RANDOM: JSON.stringify(true),
     }),
+    new webpack.ProvidePlugin({
```

Comment:
Copied from the core

https://github.com/mui/material-ui/blob/c8feb29320a22cc4461fb5ef5f4f7de7bd96a59b/test/regressions/webpack.config.js#L26
I think the Storybook was injecting a default config somehow.